File: order.py
class Order:
    def __init__(self, name, products, totalPrice, date):
        self.name = name
        self.products = products
        self.totalPrice = totalPrice
        self.date = date

    def printProducts(self):
        for product in self.products:
            print("\t" + product)

    def getName(self):
        return self.name

    def getProducts(self):
        return self.products

    def getTotalPrice(self):
        return self.totalPrice

    def getDate(self):
        return self.date

File: clients/Order.java
package clients;

import java.util.ArrayList;

public class Order {
    private String name;
    private ArrayList<String> products;
    private double totalPrice;
    private String date;

    public Order(String name, ArrayList<String> products, double totalPrice, String date) {
        this.name = name;
        this.products = products;
        this.totalPrice = totalPrice;
        this.date = date;
    }
    public void printProducts(){
        for (String product : products){
            System.out.println("\t"+product);
        }
    }

    public String getName() {
        return name;
    }

    public ArrayList<String> getProducts() {
        return products;
    }

    public double getTotalPrice() {
        return totalPrice;
    }

    public String getDate() {
        return date;
    }
}
